org.eclipse.jdt.core.javac/src/org/eclipse/jdt/core/dom/JavacConverter.java

```diff
@@ -918,19 +940,20 @@ private VariableDeclaration convertVariableDeclaration(JCVariableDecl javac) {
 		SingleVariableDeclaration res = this.ast.newSingleVariableDeclaration();
 		commonSettings(res, javac);
 		if (convertName(javac.getName()) instanceof SimpleName simpleName) {
-			int endPos = javac.getEndPosition(this.javacCompilationUnit.endPositions);
-			if( !simpleName.toString().equals(FAKE_IDENTIFIER)) {
-				char theChar = this.rawText.charAt(endPos);
-				char soughtLastChar = simpleName.toString().charAt(simpleName.toString().length() - 1);
-				while (endPos > res.getStartPosition() && theChar != soughtLastChar) {
-					theChar = this.rawText.charAt(--endPos);
-				}
-				endPos++;
-				int length = simpleName.toString().length();
-				if( endPos != -1 && endPos - length > 0) {
-					simpleName.setSourceRange(endPos - length, length);
-				}
-			}
+			// int endPos = javac.getEndPosition(this.javacCompilationUnit.endPositions);
+			// if( !simpleName.toString().equals(FAKE_IDENTIFIER)) {
+			// 	char theChar = this.rawText.charAt(endPos);
+			// 	char soughtLastChar = simpleName.toString().charAt(simpleName.toString().length() - 1);
+			// 	while (endPos > res.getStartPosition() && theChar != soughtLastChar) {
+			// 		theChar = this.rawText.charAt(--endPos);
+			// 	}
+			// 	endPos++;
+			// 	int length = simpleName.toString().length();
+			// 	if( endPos != -1 && endPos - length > 0) {
+			// 		simpleName.setSourceRange(endPos - length, length);
+			// 	}
+			// }
```

Comment:
Please just remove

Reply:
I will remove it once all tests passes as the final step, Until I will keep this review open as a reminder.